src/components/v5/frame/ColonyHome/partials/ReputationChart/partials/TeamActionsMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import { DotsThreeVertical, Eye, Lock, Pencil } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import { Link } from 'react-router-dom';
+
+import { Action } from '~constants/actions.ts';
+import { useActionSidebarContext } from '~context/ActionSidebarContext/ActionSidebarContext.ts';
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import { tw } from '~utils/css/index.ts';
+import { formatText } from '~utils/intl.ts';
+import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts.ts';
+
+const displayName =
+  'v5.frame.ColonyHome.ReputationChart.partials.TeamActionsMenu';
+
+const MSG = defineMessages({
+  viewAllTeams: {
+    id: `${displayName}.viewAllTeams`,
+    defaultMessage: 'View all teams',
+  },
+  createTeam: {
+    id: `${displayName}.createTeam`,
+    defaultMessage: 'Create new team',
+  },
+  editTeam: {
+    id: `${displayName}.editTeam`,
+    defaultMessage: 'Edit team',
+  },
+});
+
+// @TODO maybe decouple some of this into a dropdown menu component?
+// eslint-disable-next-line max-len
+const dropdownItemClassName = tw`flex w-full items-center gap-2 rounded-s px-3 py-2 text-md text-gray-900 hover:font-medium md:hover:bg-gray-50`;
+
+const TeamActionsMenu = () => {
+  const {
+    colony: { name: colonyName },
+  } = useColonyContext();
+  const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
+  const { getTooltipProps, setTooltipRef, setTriggerRef, visible } =
+    usePopperTooltip({
+      placement: 'bottom-end',
+      trigger: ['click'],
+      visible: isDropdownOpen,
+      onVisibleChange: setIsDropdownOpen,
+      interactive: true,
+      closeOnOutsideClick: true,
+      offset: [0, 0],
+    });
+
+  const {
+    actionSidebarToggle: [, { toggleOn: toggleActionSidebarOn }],
+  } = useActionSidebarContext();
+
+  const handleCreateTeamClick = () => {
+    toggleActionSidebarOn({
+      [ACTION_TYPE_FIELD_NAME]: Action.CreateNewTeam,
+    });
+    setIsDropdownOpen(false);
+  };
+
+  const handleEditTeamClick = () => {
+    toggleActionSidebarOn({
+      [ACTION_TYPE_FIELD_NAME]: Action.EditExistingTeam,
+    });
+    setIsDropdownOpen(false);
+  };
+
+  return (
+    <>
+      <button
+        type="button"
+        ref={setTriggerRef}
+        className={clsx('hover:text-blue-400', {
+          'text-gray-600': !visible,
```

Comment:
This needs to be color `gray-400` in its non-hovered/non-active state based on the [Figma designs](https://www.figma.com/design/gac6xVgGNCAge5Bia933dp/Dashboard?node-id=4878-76054&t=27Zn8LxD7w4sTlH7-4)

```js
className={clsx('text-gray-400 hover:text-blue-400', {
  '!text-blue-400': visible,
})}
```

![image](https://github.com/user-attachments/assets/2fa418e6-f402-4ca0-aced-4d2fc6c6589b)